recipes/repeated-test/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "repeated_test" %}
+{% set version = "1.0" %}
+{% set sha256 = "73193f2bb360051ec78faeadc72b8b70240852e3f2746ff1da1526f4c03be33a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.7
+
+  run:
+    - python
+    - six >=1.7
+
+test:
+  imports:
+    - repeated_test
+    - repeated_test.tests
+
+  requires:
+    - unittest2  # [py26]
+
+about:
+  home: https://github.com/epsy/repeated_test
+  license: MIT
+  license_family: MIT
+  # license_file: LICENSE.txt # Waiting on new release
```

Comment:
Is this why it is WIP? If so, I'd rather just merge and save that for the feedstock. Though am not going to push it if you would rather wait. What do you think?